content/en/docs/topics/kubernetes_apis.md

```diff
@@ -0,0 +1,127 @@
+for deprecation states the timeframe that API versions will be supported following
+a deprecation announcement. It is therefore important to be aware of deprecation
+announcements and know when API versions will be removed, to help mimimize the
+effect.
+
+This is an example of an announcement [for the removal of deprecated API versions in Kubernetes 1.16](https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/)
+and was advertised a few months prior to the release. These API versions would
+have been announced for deprecation prior to this again. This shows that there
+is a good policy in place which informs consumers of API version support. 
+
+Helm templates specify a [Kubernetes API group](https://kubernetes.io/docs/concepts/overview/kubernetes-api/#api-groups)
+when defining a Kubernetes object, similar to a Kubernetes manifest file. It is
+specified in the `apiVersion` field of the template and it identifies the API
+version of the Kubernetes object. This means that Helm users and chart maintainers
+need to be aware when Kubernetes API versions have been deprecated and in what
+Kubernetes version they will be removed.
+
+## Chart Maintainers
+
+You should audit your charts checking for Kubernetes API versions that are
+deprecated or are removed in a Kubernetes version. The API versions found out of
+support should be updated to the supported version and a new version of the chart
+released. The API version is defined by the `kind` and `apiVersion` fields. For
+example, here is a removed `Deployment` object API version in Kubernetes 1.16:
+
+```yaml
+apiVersion: apps/v1beta1
+kind: Deployment
+```
+
+## Helm Users
+
+You should audit the charts that you use (similar to the [chart mantainer](#chart-maintainers))
+and identify any charts where API verions are deprecated or removed in a
+Kubernetes version. For the charts identified, you need to check for the latest
+version of the chart (which has supported API versions) or update the chart yourself.
+
+Additionally, you also need to audit any charts deployed (i.e. Helm releases) checking
+again for any deprecated or removed API versions. This can be done by getting
+details of a release using the `helm get` command.
+
+The means for updating a Helm release to supported APIs depends on your findings
+as follows:
+
+1. If you find deprecated API versions only then:
+  - Perform a `helm upgrade` with a version of the chart with supported Kubernetes
+  API versions
+  - Add a description in the upgrade, something along the lines to not perform a
+  rollback to a Helm version prior to this current version
+2.  If you find any API version(s) that is/are removed in a Kubernetes version then:
+  - If you are on a Kubernetes version where the removed API version(s) are in a
+  later Kubernetes version (i.e. you are on Kubernetes 1.15 but the removed API
+  versions are in Kubernetes 1.16):
+    - Follow the step 1 procedure
+  - Otherwise:
```

Comment:
To clarify how about:
"Otherwise (i.e., you are already running a Kubernetes version where some API versions reported by `helm get manifest` are no longer available):"

Reply:
Better, updated.